modules/openedx/application/security-groups.tf

```diff
@@ -0,0 +1,62 @@
+locals {
+  allow_prometheus_0_to_65535_ingress_groups = [
+    aws_security_group.director,
+    aws_security_group.edxapp_appserver
+  ]
+
+  allow_prometheus_0_to_65535_ingress_group_map = {
+    for sg in local.allow_prometheus_0_to_65535_ingress_groups :
+    sg.name => sg.id
+  }
+}
+
+resource "aws_security_group" "allow_prometheus" {
+  // TODO: change name and description
+  name        = "allow_prometheus"
```

Comment:
@Cup0fCoffee Change this to use the same naming convention being used on this folder.